src/components/FocusTrap/FocusTrapForModal/index.web.tsx

```diff
@@ -10,7 +10,7 @@ function FocusTrapForModal({children, active}: FocusTrapForModalProps) {
             focusTrapOptions={{
                 trapStack: sharedTrapStack,
                 allowOutsideClick: true,
-                fallbackFocus: document.body,
+                initialFocus: false,
```

Comment:
Since we did not have a configurable `initialFocus`, we ran into initial focus issues for AttachmentModal in #43726

Reply:
Since we removed initial focus and fallbackFocus on document body, it introduced this bug on mSafari: https://github.com/Expensify/App/issues/50661

More details in here: https://github.com/Expensify/App/issues/50661#issuecomment-2422967440